language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/document/BallerinaDocumentService.java

```diff
@@ -403,6 +403,7 @@ public CompletableFuture<PackageMetadataResponse> project(BallerinaProjectParams
                 if (project.isEmpty()) {
                     return ballerinaProject;
                 }
+                ballerinaProject.setOrgName(project.get().currentPackage().packageOrg().value());
```

Comment:
Since this project service is deprecated from ballerina 2.0.0 there arn't any tests.